config/default.yaml

```diff
@@ -8,7 +8,7 @@ proxy:
 idam:
   base_url: http://localhost:4501
   s2s_url: http://localhost:4502
-  service_key:
+  service_key: AAAAAAAAAAAAAAAC
```

Comment:
Do we need to change that?

Reply:
Not mandatory, but it's good to have it sync with `ccd-docker` 👍

Reply:
Agree 👍